rainbows.rb

```diff
@@ -25,5 +26,21 @@ def mix_colors(color1,color2)
   return mixed_color
 end
 
+rainbow = [{ red: { r:255,g:0,b:0 }},
+          { orange: { r:255, g:127, b:0 }},
+          { yellow: { r:255, g:255, b:0 }},
+          { green: { r:0, g: 255, b:0 }},
+          { blue: { r:0, g: 0, b:255 }},
+          { violet: { r: 125, g:0, b:255 }},
+          { magenta: { r:255, g:0, b:255 }}]
+
+def print_rainbow(array)
+  array.each do |array|
+    puts array.keys
+  end
+end
```

Comment:
This is very strange that array.each and |array| use the same variable names. I guess ruby is smart enough to figure out which one you are referring to, but I think |array| should be |element| or something similar.